commown_self_troubleshooting/data/Assistance/Generic Issue/issue_template.xml

```diff
@@ -23,19 +23,29 @@
           </t>
           <p>
             L'adresse d'expédition est la suivante : <br/>
-            <t t-esc="name"/><br/>
-            <t t-esc="u'\n'.join((street, street2)).strip()"/><br/>
-            <t t-esc="zipcode"/>
-            <t t-esc="city"/> <br/>
-            <t t-esc="country"/><br/>
+            <ul>
+              <li t-esc="name"/>
+              <li t-esc="street"/>
+              <li t-esc="street2"/>
```

Comment:
street2 is pretty often empty (and maybe other fields could?) so I would rather write:
`<li t-if="street2.strip()" t-esc="street2.strip()"/>`